keyboards/treadstone48/config.h

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+// GCC include 'config.h" sequence in qmk_firmware/keyboards/treadstone48/
+//   -include keyboards/treadstone48/config.h
+//   -include keyboards/treadstone48/rev?/config.h
+//   -include keyboards/treadstone48/rev?/keymaps/MAPNAME/config.h
+//   XXXX.c
+
+#include <serial_config.h>
+
+// GCC include search path in qmk_firmare/keyboards/treadstone48/
+//    #include "..." search starts here:
+//    #include <...> search starts here:
+//     keyboards/treadstone48/rev?/keymaps/MAPNAME
+//     keyboards/treadstone48
+//     keyboards/treadstone48/rev?
+//     .
+//     ./tmk_core
+//     ......
+
+#ifdef USE_Link_Time_Optimization
```

Comment:
Unless you're adding the `OPT_DEFS` to add this, this won't get defined. 

Additionally, MACRO and FUNCTION are features that are depreciated, so disabling them by default isn't a bad option, anyways. 

```suggestion
```